doc/getting-started-guide/faq.rst

```diff
@@ -186,6 +186,136 @@ What other projects leverage xarray?
 
 See section :ref:`ecosystem`.
 
+How can I read format X in xarray?
+----------------------------------
+
+To read format X in xarray, you need to know the `format of the data <https://docs.xarray.dev/en/stable/user-guide/io.html#csv-and-other-formats-supported-by-pandas/>`_ you want to read. If the format is supported, you can use the appropriate IO function provided by xarray. The following table provides links to IO functions for different file formats in xarray, as well as links to other packages that can be used to read these formats:
+
+.. csv-table::
+   :header: "File Format", "xarray IO Function", " Other Packages"
+   :widths: 15, 35, 15
+
+   "NetCDF (.nc, .nc4, .cdf)","xarray.open_dataset() OR xarray.open_mfdataset()", "`netCDF4 <https://pypi.org/project/netCDF4/>`_, `netcdf <https://pypi.org/project/netcdf/>`_ , `cdms2 <https://cdms.readthedocs.io/en/latest/cdms2.html>`_"
+   "HDF5 (.h5, .hdf5)","xarray.open_dataset() OR xarray.open_mfdataset()", "`h5py <https://www.h5py.org/>`_, `pytables <https://www.pytables.org/>`_ "
+   "GRIB1/GRIB2 (.grb, .grib)", "xarray.open_dataset()", "`cfgrib <https://pypi.org/project/cfgrib/>`_, `pygrib <https://pypi.org/project/pygrib/>`_"
+   "Zarr","xarray.open_zarr()","`zarr <https://zarr.readthedocs.io/en/stable/>`_ , `fsspec <https://filesystem-spec.readthedocs.io/en/latest/>`_"
+   "CSV (.csv)","xarray.open_dataset()<br>xarray.open_mfdataset()","`pandas <https://pandas.pydata.org/>`_ , `dask <https://www.dask.org/>`_ "
+   "Excel (.xls, .xlsx)","xarray.open_dataset()","`pandas <https://pandas.pydata.org/>`_, `openpyxl <https://pypi.org/project/openpyxl/>`_ "
+   "JSON (.json)","xarray.open_dataset()","`json <https://docs.python.org/3/library/json.html>`_, `pandas <https://pandas.pydata.org/>`_"
+
+To use these IO functions in xarray, you can simply call them with the path to the file(s) you want to read as an argument.
```

Comment:
The main thing you're missing here is the idea of passing a different `engine` to `xr.open_dataset`. That's what actually tells xarray what file format you are attempting to load. A lot of the entries in your list below should really include the `engine` keyword argument, e.g: 
`ds = xr.open_dataset('/path/to/file.zarr', engine='zarr')`

(There are some weird exceptions like how `open_zarr` also exists, but we're possibly going to get rid of those, see https://github.com/pydata/xarray/issues/7495.)

In order for the `engine` kwarg to work, the corresponding xarray file backend has to be registered, which is normally done by installing some other package. For example I don't think you need to `import cfgrib` explicitly to use `engine='cfgrib' ([see example](https://github.com/ecmwf/cfgrib#usage)), but you do need to have the cfgrib package installed. We should mention this.